app/views/events/_participants_modal.html.erb

```diff
@@ -23,7 +23,10 @@
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-default" data-dismiss="modal"><%= t('.close_modal') %></button>
-        <%= submit_tag(t('.create_pdf'), class: 'btn btn-default', id:'print_participant_list' ) %>
+        <%= submit_tag(t('.create_pdf'), 
+            class: 'btn btn-default', 
+            id:'print_participant_list',
+            :formtarget => "_blank" ) %>
```

Comment:
Why do you use `formtarget` here and `target` in the other `submit_tag`?
Also, mixing the two syntaxes for hashes looks weird, please keep it consistent.